### ISSUE TITLE
Notification with role="alert" is not announced after performing an action in a dialog

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#200]
**Describe the bug**
After performing an action in a dialog, the notification that appears confirming the action is not read back by the screen reader. Currently this can be seen with actions related to saved searches: saving a search, and deleting a saved search.

Tested with NVDA.

**To Reproduce**

1. Perform a search
2. Turn on a screen reader (i.e. NVDA)
3. Select "Save my search"
4. Select "Save"
5. Notice that the notification is not read back by the screen reader.

**Expected behavior**
After performing an action in a dialog, the notification should be read back with the screen reader.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox + NVDA

**Comment by jhung** [comment]
Could not confirm whether this happens in Pinecone as the Search Results layout does not produce a notification.

Also, could it be that the notification is being added to the document while it is inert? Perhaps removing inert before adding the notification will trigger the desired response from the screen reader.

**Comment by greatislander** [comment]
Hi @jhung, I can't reproduce this in Safari/VoiceOver. The notification has `role="alert"`. Do you know if NVDA needs `aria-live` as well to read elements added to the DOM? The notification is not being added to the DOM while it's inert (or at any rate it's not preventing VoiceOver from properly announcing it).
